public/js/util/utils.js

```diff
@@ -97,6 +97,12 @@ function entries(obj) {
   return Object.keys(obj).map(k => [k, obj[k]]);
 }
 
+function mapObject(obj, iteratee) {
```

Comment:
Sounds good to me. I'm also fine if you want to pull in a few lodash methods eventually. I'd rather not depend on a huge amount of lodash, but some of its methods might be helpful. (I know I was more resistant before.) This is good for now.